### DISCUSSION TITLE
Confyui cannot be acced from remotely with gradio url

### DISCUSSION THREAD

**Discussion by imperioame** [ModelSurge/sd-webui-comfyui#138] [Q&A] [answered]
Hi! I want to fully test confyui alongside A1111 before deciding for one or another. Hence i'm in this repo.
The thing is, I have A1111 seted up in a desktop in my house that generates a gradio url (secured) everytime turn it on.
I rarely use the server when in my house. I'm usually remote and A1111 works well with this.
Today I installed confyui extension and it's giving me a 'conection rejected' error when i try to acces it.
I know it's something regarding this remote access. Anyone know how to fix it?

Remote access with gradio url
![image](https://github.com/ModelSurge/sd-webui-comfyui/assets/10353593/68b5ab10-bc57-47e0-ac27-42c016988470)

local access with same gradio url
![image](https://github.com/ModelSurge/sd-webui-comfyui/assets/10353593/a86d6986-1547-472e-8d71-c619692b867e)

**Comment by ljleb** [comment]
We are aware of the issue, see #107, #82, #9. I think a good solution would be to setup a reverse proxy from the webui to comfyui. Then, point the iframes to the reverse proxy on a1111 api. This would make it so that there is no need to manually setup an additional external reverse proxy for comfyui.

**Reply by yurashkurenko** [reply]
Nginx proxy manager

**Reply by ljleb** [reply, marked as answer]
Forgot to come back to this. The reverse proxy option was added in the options of the extension last year. I intends to resolve specifically the problem mentioned above among other situations.

**Reply by ayush1268** [reply]
> Forgot to come back to this. The reverse proxy option was added in the options of the extension last year. I intends to resolve specifically the problem mentioned above among other situations.

can you help me a bit acctually i wanted to run comfyui in lighting ai studio llabs but the local host address not work ofcourse, acctaully i wanted to know how can i use gradio link in my comfyui to launch it
